src/deferred/exceptionHook.js

```diff
@@ -7,14 +7,20 @@ define( [
 
 // These usually indicate a programmer mistake during development,
 // warn about them ASAP rather than swallowing them by default.
-var rerrorNames = /^(Eval|Internal|Range|Reference|Syntax|Type|URI)Error$/;
+// Notice that Error is a choice too, as ? regex means (0-1) times.
```

Comment:
We excluded `Error` on purpose. The idea is that we want to log only those errors that are most likely the result of a simple user error (like a mistyped variable) rather than all errors.